docs/NMOS With JPEG XS.md

```diff
@@ -46,24 +71,46 @@ Sender resources provide no indication of media type or format, since this is de
 The SDP file at the `manifest_href` MUST comply with the requirements of RFC 9134.
 If the Sender meets the traffic shaping and delivery timing requirements specified for ST 2110-22, the SDP file MUST also comply with the provisions of ST 2110-22.
 
-### Flows
+For Nodes implementing IS-04 v1.3 or higher, the following additional requirements on the Sender resource apply.
 
-The Flow resource MUST indicate `video/jxsv` in the `media_type` attribute, and `urn:x-nmos:format:video` for the `format`.
+In addition to those attributes defined in IS-04 for Senders, the following attributes defined in the [Sender Attributes register](https://specs.amwa.tv/nmos-parameter-registers/branches/main/sender-attributes/) of the NMOS Parameter Registers are used for JPEG XS.
+
+- [Bit Rate](https://specs.amwa.tv/nmos-parameter-registers/branches/main/sender-attributes/#bit-rate)  
+  If the Sender complies with ST 2110-22, the Sender resource MUST indicate the bit rate (kilobits/second) including the RTP transport overhead.
+  Otherwise, it is only RECOMMENDED that this attribute is included.
+  The `bit_rate` integer value is expressed in units of 1000 bits per second, rounding up.
+  The value is for the IP packets, including the RTP, UDP and IP packet headers and the payload.
```

Comment:
Could we formulate such as to already support other transports like NDI and SRT ... We could refer to "The value is for IP packets encapsulated into their specific transport. For example for the RTP transport type it includes the RTP, UDP and IP packet headers and the payload.

Reply:
Yes, this is a good idea

Reply:
```suggestion
  The value is for the IP packets, so for the RTP payload format per RFC 9134, that includes the RTP, UDP and IP packet headers and the payload.
```